src/components/MovieInfo.tsx

```diff
@@ -13,11 +15,13 @@ export type DataChildren = {
 };
 
 const MovieInfo: React.FC<MovieInfoProps> = ({ data }) => {
+
   return (
     <div className="container">
       <div className="row justify-content-between align-items-center">
-        {data.map((elem, i) => {
-          const {Poster, Year, Title} = elem
+        {data.map((item, i) => {
+          const { Title, Year, Type, Poster } = item
+          const icon = Type === 'movie' ? <BiCameraMovie /> : <PiTelevisionSimple />
           return (
             <div className="col-4">
```

Comment:
In the map return of this compoent, please remove the key={i} from img tag and add it instead to its parent like this:
```
   <div key={i} className="col-4">
              <div className="card text-white bg-dark mb-3 m-2">
                <img className="card-img-top" src={Poster} alt={Title} />
                <div className="card-body">
                  <div className="card-title lead text-uppercase">{Title}</div>
                  <p className="card-text">Released on: {Year}</p>
                  <p className="card-text">
                    {icon} {Type}
                  </p>
                </div>
              </div>
            </div>
```